Fix build with modular-xorg-server-1.20

Downloaded from
http://cvsweb.netbsd.org/bsdweb.cgi/pkgsrc/x11/xf86-video-savage/patches/patch-src_savage__driver.c?rev=1.3&content-type=text/x-cvsweb-markup

Patch was sent upstream by Thomas Klausner <wiz@netbsd.org>:
https://lists.x.org/archives/xorg-devel/2018-May/056952.html

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

--- a/src/savage_driver.c.orig	2017-01-17 22:43:40.000000000 +0000
+++ b/src/savage_driver.c
@@ -2034,8 +2034,10 @@ static Bool SavagePreInit(ScrnInfoPtr pS
     xf86DrvMsg(pScrn->scrnIndex, X_PROBED, "Detected current MCLK value of %1.3f MHz\n",
 	       mclk / 1000.0);
 
+#if 0
     pScrn->maxHValue = 2048 << 3;	/* 11 bits of h_total 8-pixel units */
     pScrn->maxVValue = 2048;		/* 11 bits of v_total */
+#endif
     pScrn->virtualX = pScrn->display->virtualX;
     pScrn->virtualY = pScrn->display->virtualY;
 
